libraries/networking/src/FileResourceRequest.cpp

```diff
@@ -18,17 +18,15 @@ void FileResourceRequest::doSend() {
     
     QFile file(filename);
     
-    _state = Finished;
-    if (file.exists()) {
-        if (file.open(QFile::ReadOnly)) {
-            _data = file.readAll();
-            _result = ResourceRequest::Success;
-        } else {
-            _result = ResourceRequest::AccessDenied;
-        }
+    if (!file.exists()) {
+        _result = NotFound;
+    } else if (file.open(QFile::ReadOnly)) {
+        _data = file.readAll();
+        _result = Success;
     } else {
-        _result = ResourceRequest::NotFound;
+        _result = AccessDenied;
     }
```

Comment:
Yeah the only reason I have that change is because we both touched that code and when I rebased onto @birarda's branch I kept my changes.
I don't mind changing it.